src/Spryker/Client/RabbitMq/RabbitMqConfig.php

```diff
@@ -73,13 +70,39 @@ protected function getQueueConnectionConfigs()
         return $connections;
     }
 
+    /**
+     * @return \ArrayObject
+     */
+    protected function getQueueOptions()
```

Comment:
return type

Reply:
This method is overridden for all projects. If I add type-hint here I will break them.